jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -2256,17 +2256,38 @@ public boolean hasFullPrivilegeAccess() {
 		}
 
 		static class ClassDefiner {
-			Class<?> defineClass(boolean option) {
-				return null;
+			private byte[] classBytes;
+			private String className;
+			private Lookup lookup;
+			ClassDefiner(String name, byte[] template, Lookup lookupObj) {
+				className = name;
+				classBytes = template;
+				lookup = lookupObj;
+			}
+			Class<?> defineClass(boolean initOption) {
+				JavaLangAccess jlAccess = SharedSecrets.getJavaLangAccess();
+				Class<?> lookupClass = lookup.lookupClass();
+				return jlAccess.defineClass(lookupClass.getClassLoader(), lookupClass, className, classBytes, jlAccess.protectionDomain(lookupClass), initOption, 0, null);
 			}
 		}
 
 		public Lookup defineHiddenClass(byte[] bytes, boolean initOption, ClassOption... classOptions) {
-			return null;
+			ClassReader cr = null;
```

Comment:
Please remove the unnecessary initialization.

Reply:
Fixed.